### PR TITLE
Crafting: Spinning bowstrings, wool etc

### PR DESCRIPTION
**Pull request by Promises**

### CODE REVIEW

File: src/plugins/skills/crafting/spinning-wheel-plugin.ts [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    if (!details.player.activeWidget || !(details.player.activeWidget.widgetId === 459)) {
+        return;
+    }
+    const product = widgetButtonIds.get(details.buttonId);
+
+    // Close the widget as it is no longer needed
+    details.player.closeActiveWidget();
+
+    if (!details.player.skills.hasSkillLevel(Skill.CRAFTING, product.spinnable.requiredLevel)) {
+        details.player.outgoingPackets.chatboxMessage(`You need a crafting level of ${product.spinnable.requiredLevel} to craft this item.`);
```
Comment (TheBlackParade) [inline src/plugins/skills/crafting/spinning-wheel-plugin.ts]: Try `player.sendMessage(message, true);` :)

File: src/plugins/skills/crafting/spinning-wheel-plugin.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                    currentItemIndex++;
+                    currentItem = (<number[]> spinnable.input)[currentItemIndex];
+                } else {
+                    cancel = true;
+                }
+            } else {
+                cancel = true;
+            }
+            if (cancel) {
+                details.player.outgoingPackets.chatboxMessage(`You don't have any ${gameCache.itemDefinitions.get(currentItem).name.toLowerCase()}.`);
```
Comment (TheBlackParade) [inline src/plugins/skills/crafting/spinning-wheel-plugin.ts]: `player.sendMessage(message, true)`

Comment (Promises) [reply src/plugins/skills/crafting/spinning-wheel-plugin.ts]: for this one its just sent in the chatbox in osrs.

Comment (TheBlackParade) [inline src/plugins/skills/crafting/spinning-wheel-plugin.ts]: Don't forget this one :d

**Pull request merged**